qemu/tests/qemu_guest_agent.py

```diff
@@ -820,6 +851,24 @@ class QemuGuestAgentBasicCheckWin(QemuGuestAgentBasicCheck):
     """
     Qemu guest agent test class for windows guest.
     """
+
+    @error_context.context_aware
+    def gagent_uninstall(self, test, session, vm, *args):
+        if args and isinstance(args, tuple):
+            gagent_uninstall_cmd = args[0]
+        else:
+            test.error("Missing config 'gagent_uninstall_cmd'")
+
+        if not gagent_uninstall_cmd:
+            return
```

Comment:
IMO, this command must be specified if you do want to uninstall the package, so I'd suggest reporting `test.error` here.

Reply:
Modified.
Thanks.